landing-page/src/ha-landing-page.ts

```diff
@@ -88,35 +106,78 @@ class HaLandingPage extends LandingPageBaseElement {
     }
     import("../../src/components/ha-language-picker");
 
-    this._scheduleCoreCheck();
+    this._pingSupervisor();
+  }
+
+  private _schedulePingSupervisor() {
+    setTimeout(
+      () => this._pingSupervisor(),
+      SCHEDULE_FETCH_NETWORK_INFO_SECONDS * 1000
+    );
+  }
+
+  private async _pingSupervisor() {
+    try {
+      const response = await pingSupervisor();
+      if (!response.ok) {
+        throw new Error("Failed to ping supervisor, assume update in progress");
+      }
+      this._fetchSupervisorInfo(true);
+    } catch (_err) {
+      this._schedulePingSupervisor();
+    }
   }
 
-  private _scheduleCoreCheck() {
+  private _scheduleFetchSupervisorInfo() {
     setTimeout(
-      () => this._checkCoreAvailability(),
-      SCHEDULE_CORE_CHECK_SECONDS * 1000
+      () => this._fetchSupervisorInfo(true),
```

Comment:
Why dont we combine this one with the supervisor ping? We need to ping the supervisor before we know if it is save to fetch its info anyway right?

Reply:
you mean combine the fetch function, always check ping and then get network info?